lib/core/widgets/share_with_friends.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:dairy_app/app/themes/theme_extensions/note_create_page_theme_extensions.dart';
+import 'package:flutter/material.dart';
+import 'package:share_plus/share_plus.dart';
+
+class ShareWithFriends extends StatelessWidget {
+  const ShareWithFriends({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final mainTextStyle = TextStyle(
+      fontSize: 16.0,
+      color: Theme.of(context)
+          .extension<NoteCreatePageThemeExtensions>()!
+          .mainTextColor,
+    );
+    final mainTextColor = Theme.of(context)
+        .extension<NoteCreatePageThemeExtensions>()!
+        .mainTextColor;
+    const appDescription =
+        "Discover diaryVault - a diary app designed to help you capture your thoughts, memories, and moments effortlessly. Available now on the Play Store!";
+    return GestureDetector(
+      onTap: (() async {
+        try {
+          const playstoreURL =
+              'https://play.google.com/store/apps/details?id=me.sankethbk.dairyapp';
+          await Share.share('$appDescription\n\n$playstoreURL');
+        } catch (e) {
+          throw Exception("Error sharing app");
+        }
+      }),
+      child: Row(
+        children: [
+          Icon(
+            Icons.share,
+            color: mainTextColor,
+          ),
+          const SizedBox(width: 10.0),
```

Comment:
Having an icon is good, but since we don't have icons for other tiles in settings, it looks odd on this one. Can you remove the Icon and SizedBox here.

Reply:
ok @SankethBK